index.js

```diff
@@ -4,13 +4,24 @@ module.exports = {
   name: 'ember-holy-futuristic-template-namespacing-batman',
 
   setupPreprocessorRegistry(type, registry) {
-    registry.add("htmlbars-ast-plugin", {
-      name: "holy-futuristic-template-namespacing-batman",
+    let pluginObj = this._buildPlugin();
+    pluginObj.parallelBabel = {
```

Comment:
Hmm, I think I’d prefer this to be in the file in lib directly instead of here. Thoughts?

Reply:
As discussed, We create a wrapper object to the plugin in lib so I have written it here.